Automattic/wp-calypso
# Desktop build: copy the static/ directory to desktop/public

The `build-desktop` task didn't copy the static assets (images, ...) from the `static/` directory to `desktop/public`, and these assets were then missing in the Desktop App.

This PR adds the `build-desktop:static` task that does the same copying that the web build does in `build-static`, only with a desktop-specific destination folder.

**How to test:**
1. Build the desktop app (`yarn run build-desktop`)
2. Open the desktop app built in `desktop/release`
3. On a site with Classic Editor enabled, verify that the Block Editor nudge image is displayed:

<img width="733" alt="classic-editor-supposed-to-look-like" src="https://user-images.githubusercontent.com/664258/90499114-6ca77f00-e149-11ea-9340-70f68934f74a.png">

Before this patch, the image was missing (and the HTTP request would return 404 code)
